website/content/docs/api-gateway/upgrade-specific-versions.mdx

```diff
@@ -0,0 +1,283 @@
+---
+layout: docs
+page_title: Upgrading Specific Versions
+description: >-
+  Consul api gateway versions v0.2.x introduced a change to namespace backend ref
+  enforcement that will potentially break any routes created with a cross namespace in
+  v0.1.0. This documents the flow for finding and remediating any potentially
+  broken routes
+---
+
+# Upgrading Specific Versions
+
+This guide explains how to upgrade a Consul API Gateway deployment running a specific version
+that requires additional steps from the standard upgrade path.
+
+Note: As of writing, v0.1.0 is the only previous release. A standardized upgrade path will be documented with future releases.
+
+
+## Consul API Gateway v0.1.0
+
+Consul API Gateway v0.2.0 introduced a breaking change that causes routes with a BackendRef defined in a different namespace to
+require a [ReferencePolicy](https://gateway-api.sigs.k8s.io/v1alpha2/references/spec/#gateway.networking.k8s.io/v1alpha2.ReferencePolicy)
+that explicitly allows traffic from the route's namespace to the BackendRef's namespace. This guide explains how to find all routes
+that require a ReferencePolicy and create a matching ReferencePolicy.
+
+
+
+### Requirements
+
+- The consul-api-gateway should be running version v0.1.0. If the version is different, follow the normal upgrade path.
+- **Optional** [jq](https://stedolan.github.io/jq/download/) is installed on the users CLI
+
+
+
+### Pre-requisites
+
+In order to follow this guide, the following conditions must be true:
+
+- You have the ability to run Kubectl CLI commands
+- Your kubectl config is already configured to point at the cluster with the installation you are upgrading
+- You have HTTPRoute.read, TCPRoute.read and ReferencePolicy.create rights on your Kubernetes cluster
+
+
+### Procedures
+
+**1.** Optional, double check the current version of the consul-api-gateway with the following command:
```

Comment:
```suggestion
**1.** Double check the current version of the consul-api-gateway-controller Deployment with the following command:
```

If we think it's worth including, I wouldn't mark as optional. I think people will naturally skip it if they have full faith in themselves.